alerting/build.gradle

```diff
@@ -98,6 +98,15 @@ integTest.dependsOn(bundle)
 integTest.getClusters().forEach{c -> c.plugin(project.getObjects().fileProperty().value(bundle.getArchiveFile()))}
 
 def _numNodes = findProperty('numNodes') as Integer ?: 1
+
+String notificationsVersion = "2.0.0"
+String notificationsQualifier = "rc1"
```

Comment:
Can we use the [opensearch_build](https://github.com/opensearch-project/alerting/blob/main/build.gradle#L15) and [buildVersionQualifier](https://github.com/opensearch-project/alerting/blob/main/build.gradle#L11) defined in our build.gradle for these values instead? These will keep changing so it'd be better to just use the source of truth.

Also, we should account for there being no qualifier since that's possible. In that case we'd omit the `"-" + notificationsQualifier` in the links below